urc_perception/multiclass_segmentation/README.md

```diff
@@ -80,7 +80,7 @@ You can alternatively use a google colab to train your model. Here is a brief in
 
 2.  Open the [ConvertImages_SplitData.ipynb](https://colab.research.google.com/drive/15T2XQjBKMh92A3XlsfR-kRX-PFVN9rl4?usp=sharing) Google Colab Notebook and follow the steps to convert your dataset into the appropriate format and create train/test splits.
     
-3.  Open the [urc_multiunet.ipynb](https://colab.research.google.com/drive/1bb9TRCNWBgV8-EiqzhjlSQM_yny9DXzt?usp=sharing) Google Colab Notebook and follow the steps to train the network. Make sure to adjust the config file in igvc_perception/src/multiclass_segmentation/config/igvc.yaml for desired training parameters and to specify model save location.
+3.  Open the [urc_multiunet.ipynb](https://colab.research.google.com/drive/1bb9TRCNWBgV8-EiqzhjlSQM_yny9DXzt?usp=sharing) Google Colab Notebook and follow the steps to train the network. Make sure to adjust the config file in urc_perception/multiclass_segmentation/config/urc.yaml for desired training parameters and to specify model save location.
```

Comment:
Why is multiclass_segmentation not in a src folder? Shouldnt it follow the same format as all other nodes?

Reply:
I personally believe that multiclass segmentation should be it's own directory because it is more than just the one node. It has its own README, lots of training files, seperate config files, etc. In my opinion anything that complex really deserves its own directory, I view `src` as a holder for more prototypical nodes.

Reply:
Ok sounds good. Just wanted to clarify this point.